src/main/java/com/ericsson/ei/controller/FailedNotificationController.java

```diff
@@ -10,22 +10,22 @@
 
 
 /**
- * This class searches for missed notifications in the database.
+ * This class searches for failed notifications in the database.
  * (Generated with springmvc-raml-parser v.2.0.4)
  * 
  */
 @RestController
 @Validated
-@RequestMapping(value = "/queryMissedNotifications", produces = "application/json")
-public interface QueryMissedNotificationController {
+@RequestMapping(value = "/failedNotifications", produces = "application/json")
+public interface FailedNotificationController {
 
 
     /**
-     * This method returns missed notification(s) given a subscription name.
+     * This method returns failed notification(s) given a subscription name.
      * 
      */
     @RequestMapping(value = "", method = RequestMethod.GET)
-    public ResponseEntity<?> getQueryMissedNotifications(
+    public ResponseEntity<?> getFailedNotifications(
         @RequestParam
         String subscriptionName);
```

Comment:
The parameter should be updated to be plural, to align with the /subscriptions endpoint. subscriptionName --> subscriptionNames

Reply:
done